### PR TITLE
FIKOU MACRO

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

image
## Why It's Good For The Game
Line count's lower, must be a good thing

## Changelog

:cl:
refactor: fikou macro, fikou macro, fikou macro!
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
Fikou macro :(

**Comment by tralezab** [comment]
?????

**Comment by Fikou** [comment]
Merge this. Now.

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -113,30 +113,15 @@
 			return TRUE
 	return FALSE
 
+#define FIKOU_MACRO(check_in,mob,check_against,damage,armour_penetration,attack_text,attack_type) if(check_in){var/final_block_chance=check_in.block_chance-(clamp((armour_penetration-check_in.armour_penetration)/2,0,100))+round(damage/-3);if(check_in.hit_reaction(src,AM,attack_text,final_block_chance,damage,attack_type)){return TRUE;}}
```
Comment (Watermelon914) [inline code/modules/mob/living/carbon/human/human_defense.dm:116]: fikou macro

**Pull request closed without merging**